assets/src/edit-story/components/canvas/canvasLayout.js

```diff
@@ -18,7 +18,7 @@
  * External dependencies
  */
 import styled from 'styled-components';
-import { useRef } from 'react';
+import React, { useRef } from 'react';
```

Comment:
Could you pls just bake `memo` into `{}` in import? Here and in other places.

Reply:
Done.